.github/workflows/autoformat.yml

```diff
@@ -0,0 +1,23 @@
+# Runs auto-formatting script on push to any branch
+name: "Run auto formatting"
+
+permissions:
+  id-token: write
+  contents: read
+
+on: push
+
+jobs:
+  autoformat:
+    name: autoformat
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GHA_PLATFORM_ECR_READ_ROLE_ARN }}
+      - uses: aws-actions/amazon-ecr-login@v2
+        env:
+          AWS_REGION: eu-west-1
```

Comment:
```suggestion
          AWS_REGION: us-east-1
```

Reply:
I think you may need to authorize in this region, see https://github.com/aws-actions/amazon-ecr-login?tab=readme-ov-file#configure-credentials.

This is what the working workflow in wellcomecollection.org does too: https://github.com/wellcomecollection/wellcomecollection.org/blob/main/.github/workflows/prismic-linting.yml#L21C1-L21C32